tests/yast2_gui/yast2_control_center.pm

```diff
@@ -294,6 +294,7 @@ sub start_wake_on_lan {
 sub start_directory_server {
     search 'directory server';
     assert_and_click 'yast2_control-center_authentication-server';
+    send_key 'alt-o' if (check_screen('deprecated_info', 60));
```

Comment:
@tinawang123 is it a new design for sle15sp6+ only? can you please add a `record_info` here if possible? :) anyway LGTM